files/build_templates/buffers_config.j2

```diff
@@ -190,6 +208,26 @@ def
     },
 {% endif %}
 
+{% if voq_chassis %}
+     "BUFFER_QUEUE": {
```

Comment:
Why is only the BUFFER_QUEUE changed? What about the BUFFER_PG?

Reply:
BUFFER_PG is ingress only whereas BUFFER_QUEUE is VoQ based for chassis. Due to this, It is going to be system port based instead of port based. On LC, we will apply Queue profile on local ports as well as remote ports.